config.sh

```diff
@@ -69,13 +85,34 @@ function pre_build {
         build_freetype
     fi
 
+    if [ -z "$IS_OSX" ]; then
+        export FREETYPE_LIBS=-lfreetype
+        export FREETYPE_CFLAGS=-I/usr/local/include/freetype2/
+    fi
```

Comment:
HarfBuzz fails to find FreeType on linux without this, specify the path manually.